test/integration/run-tests.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
+# This script runs the golang integration tests in the directories underneath.
+# It should be run from the same directory it is in, or in a directory above.
+# Specify the same image prefix used (if any) with build-images.sh
+#
+# Example:
+# ./run-tests.sh --prefix=docker.io/openshift/ansible-integration- --parallel=16
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+STARTTIME=$(date +%s)
+source_root=$(dirname "${0}")
+
+prefix="${PREFIX:-openshift-ansible-integration-}"
```

Comment:
That case is when we're building the images separately and pushing them to a shared registry. I wrote this originally expecting that's what we would do, then I learned the shared registry isn't ready yet. Assuming that becomes available, it should just require setting parameters in the Jenkins jobs.